app.rb

```diff
@@ -10,49 +10,90 @@
 
 # Use enumerables to -
 
+#warm-up
+#get the first apartment
+# puts apartment[0]
+# get square foot of first apartment
+# apartment[0][:square_feet][0]
+
   # Print all the addresses for the apartments
-apartments = each.do |address|
-  puts address [:apartments]
-end
+# apartments.each do |apartment|
+#   apartments[:address]
+# end
+# puts address.inspect
+
 
   # Print all the names for tenants
-tenants = each.do |tenant|
-  puts tenants[:name]
-end
+# tenants.each do |tenant|
+#   puts tenants[:name]
+# end
 
   # Print only apartments that are less then 700 in rent
-rent = each.sort_by |monthly_rent|
- puts rent[:monthly_rent]
-end
+  # Cheap_apartments equals the monthly_rent that will be selected in the apartments. Print the apartments with monthy rents of less than 700. Show the length of the cheap_apartments. Exit the program.
+# cheap_apartments = apartments.select do |monthly_rent|
+#  puts apartments[:monthly_rent] < 700
+#  cheap_apartments.lengths
+# end
 
   # Print only tenants that are over the age of 44
-tenants = tenants.select do |tenant|
-  apartment]:age > 44
-end
-puts tenants
+  # Older_tenants represents tentants with the apartment_id of 2 that will be selected by tenants. Print older_tenants.
+# older_tenants = tenants.select { |tenant| tenant[:apartment_id] = 2 }
+#   puts older_tenants
+#   puts "*" * 50
+#   puts tenants
+
+
   # Print only tenants that have an apartment id of 1
-  apartment_one = tenants.select do |tenant|
-    tenant[:apartment_id] == 1
-  end
-  puts apartment_1
+# apartment_one = tenants.select do |tenant|
+#   tenant[:apartment_id] == 1
+# end
+# puts apartment_1
```

Comment:
What you have succesfully selects the tenants that have the apartment_id of 1 and stores them in a variable ` apartment_one`. Instead of `puts apartment_one` (which would work) you said `puts apartment_1`
Computers aren't smart enough to infer that when you said `apartment_1` you actually meant `apartment_one`, you need to be extremely careful about being consistent with how you call things